index.Rmd

```diff
@@ -32,7 +32,7 @@ In this course we will describe what the new policy requires, places where you m
 
 ## Motivation
 
-Navigating [https://sharing.nih.gov/](https://sharing.nih.gov/) can be confusing. We want to help make the process of compliance with this new policy easier. 
+Navigating [`https://sharing.nih.gov/`](https://sharing.nih.gov/) can be confusing. We want to help make the process of compliance with this new policy easier. 
```

Comment:
hmmm should we maybe reword this a tad just to make it seem like we are friendlier to the NIH :P - Like navigating the new policy?

Reply:
yes totally! good call